packages/swarm-styles/lib/overrides.css

```diff
@@ -0,0 +1,8 @@
+/*
+ * TODO: Remove this file once swarm v1 usage is removed from mup-web and pro-web
+ */
+
+.text--caption,
+.text--secondary {
+    color: var(--color-gray-7);
```

Comment:
it would be helpful for people looking at this code to know what 'override' refers to in this context - it's hard to understand what the purpose/expected behavior of this style rule is without context

Reply:
added some comments